extension/algo/src/function/spanning_forest.cpp

```diff
@@ -245,6 +250,59 @@ void KruskalCompute::mergeComponents(const offset_t& srcCompId, const offset_t&
     }
 }
 
+void KruskalCompute::getValueAsDouble(const std::shared_ptr<common::ValueVector>& valueVector,
```

Comment:
Is this also needed in PR and other algorithms? Then this function should be put in common utils somewhere.

@ray6080 is there no existing function to do this casting? It seems core operators will need something similar when reading values?